src/Abstractions/TaskFailureDetails.cs

```diff
@@ -34,7 +35,8 @@ public override string ToString()
     /// </remarks>
     /// <typeparam name="T">The type of exception to test against.</typeparam>
     /// <returns>Returns <c>true</c> if the <see cref="ErrorType"/> value matches <typeparamref name="T"/>; <c>false</c> otherwise.</returns>
-    public bool IsCausedBy<T>() where T : Exception
+    public bool IsCausedBy<T>()
+        where T : Exception
```

Comment:
I'm not sure I like this rule. I feel like it's nicer to have generic constraints on the same line as the method signature. Thoughts?

Reply:
Suppressed this rule. I like having it on a new line, but don't feel strongly about it - so we can leave it to a case by case preference.